staticaddr/withdraw/manager.go

```diff
@@ -557,11 +558,17 @@ func (m *Manager) publishFinalizedWithdrawalTx(ctx context.Context,
 	// Publish the withdrawal sweep transaction.
 	err := m.cfg.WalletKit.PublishTransaction(ctx, tx, txLabel)
 	if err != nil {
-		if !strings.Contains(err.Error(), "output already spent") &&
-			!strings.Contains(err.Error(), "insufficient fee") {
+		if !strings.Contains(err.Error(), chain.ErrSameNonWitnessData.Error()) &&
+			!strings.Contains(err.Error(), "output already spent") &&
+			!strings.Contains(err.Error(), chain.ErrInsufficientFee.Error()) {
```

Comment:
I propose to factor out this check to a function and reuse everywhere (e.g. in sweepbatcher package).

Also this commit should be squashed into commit `staticaddr: allow rbf'ing withdrawal transactions`.